src/components/CreateGroupForm/CreateGroupForm.tsx

```diff
@@ -88,18 +73,21 @@ const inputValuesReducer = (
   }
 };
 
-export default function CreateGroupForm() {
-  const [classes, setClasses] = useState<typeof query>([]);
-  // eslint-disable-next-line
-  const [filteredClasses, setFilteredClasses] = useState<typeof query>([]);
+type Props = {
+  classes: Subject[];
+};
 
+export default function CreateGroupForm({ classes }: Props) {
   const [isListOpen, setIsListOpen] = useState(false);
   const [isSubmitDisabled, setIsSubmitDisabled] = useState(true);
   const [isFormSubmitted, setIsFormSubmitted] = useState(false);
+  const [filteredClasses, setFilteredClassses] = useState<Subject[]>([]);
+  const reduxDispatch = useDispatch();
```

Comment:
I saw in JoinGroupForm that you called this dispatch. It would be nice if we could try to keep them the same. 
For a sec I thought this was completely different 😅 As someone who is not familiar with redux, I would appreciate having the same name for it

Reply:
The reason I had to do this is because for this view, we were already using a dispatch function for our error and inputValues Reducers. Both Redux and React have dispatch functions, but they cannot be used interchangeably. What I'm going to do is to keep the Redux dispatch as 'dispatch' and change the name of the others